apiservermsgs/clustermsgs.go

```diff
@@ -47,7 +47,9 @@ type CreateClusterRequest struct {
 	Name                string `json:"Name"`
 	Namespace           string
 	NodeLabel           string
-	Password            string
+	PasswordSuperuser   string
+	PasswordReplication string
+	PasswordUser        string
```

Comment:
I think we should leave this as `Password` so we don't break integrations that rely on it.

Reply:
Changed back to `Password`.